tests/expected/exceptions.dart

```diff
@@ -0,0 +1,26 @@
+// @dart=2.9
```

Comment:
tests/case/exceptions.py not included in PR

Reply:
Added now. I need to clean out my checked out git repo so there is less noise.